libkbfs/folder_branch_ops.go

```diff
@@ -3871,7 +3906,7 @@ func (fbo *folderBranchOps) UnstageForTesting(
 		// notifications if we do.  But we still want to wait for the
 		// context to cancel.
 		c := make(chan error, 1)
-		ctxWithTags := fbo.ctxWithFBOID(context.Background())
+		ctxWithTags, cancel := fbo.newCtxWithFBID()
```

Comment:
`cancel` is overridden on the next line.  I don't think we need the next line anymore actually.